google-cloud-clients/google-cloud-storage/src/test/java/com/google/cloud/storage/it/ITStorageTest.java

```diff
@@ -1661,20 +1661,16 @@ public void testGetBlobs() {
 
   @Test
   public void testDownloadPublicBlobWithoutAuthentication() {
+    assumeFalse(System.getenv("GOOGLE_CLOUD_TESTS_IN_VPCSC").equalsIgnoreCase("true"));
```

Comment:
I think this LGTM. My only request is that you make this available to future tests. Could you create a separate class member variable with this value which a test can check against?